### ISSUE TITLE
15.2.6 doesn't open on Safari 15 ( works on Safari 16)

### ISSUE THREAD

**Issue by katedoctor** [angular/angular#50148]
### Which @angular/* package(s) are the source of the bug?

bazel, compiler-cli, compiler, core

### Is this a regression?

Yes

### Description

After upgrading to 15 version of Angular, on old Safari ( v15) app doesn't work.
But it can be loaded only in case of opened devtools.
In other cases, webpack returns different issues related to compilation. 

ReferenceError: Can't find variable: _angular_common__WEBPACK_IMPORTED_MODULE_13__
Can't find variable: _angular_common__WEBPACK_IMPORTED_MODULE_12__

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI: 15.2.8
Node: 16.14.2
Package Manager: yarn 1.22.17
OS: darwin x64

Angular: 15.2.9
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... platform-server, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1502.8
@angular-devkit/build-angular   15.2.8
@angular-devkit/core            15.2.8
@angular-devkit/schematics      12.2.18
@angular/cli                    15.2.8
@schematics/angular             15.2.8
rxjs                            6.6.7
typescript                      4.8.4
webpack                         4.46.0
```

**Comment by JoostK** [comment]
This is the same issue as angular/angular-cli#24355 which was addressed in angular/angular-cli#24357, released in an Angular 15.1.x patch.

I am guessing that your browsers lists specification does not request support for Safari 15, resulting in the patch not being enabled.

**Comment by JoostK** [comment]
I am going to close this for now as it's an exact duplicate, but feel free to follow up with additional details and I may reopen if needed.
